lib/ssl/src/ssl.erl

```diff
@@ -2342,7 +2342,7 @@ opt_identity(UserOpts, #{versions := Versions} = Opts, _Env) ->
 opt_supported_groups(UserOpts, #{versions := TlsVsns} = Opts, _Env) ->
     SG = case get_opt_list(supported_groups,  undefined, UserOpts, Opts) of
              {default, undefined} ->
-                 handle_supported_groups_option(groups(default));
+                 handle_supported_groups_option(tls_v1:default_groups());
```

Comment:
I don't understand how this change can change anything.
Apart from 1 function call less ...

Reply:
Yes I think you are right, I think I misread it to do same as groups/0 due to reported symptoms. I will close this.